protocol/0056-REWA-rewards_overview.md

```diff
@@ -466,30 +466,143 @@ Market has been trading but not yet eligible for proposer bonus.
+
+#### Setup
+* Setup a market ETHUSDT settling in USDT.
+* The value of `marketCreationQuantumMultiple` is `10^6` and `quantum` for `USDT` is `1`. 
+* Setup and fund recurring reward account transfers using the market_proposer metric and `USDT` metric asset:
+    * Transfer 10000 $VEGA to `ETHUSDT | market creation | $VEGA` 
+* start trading in the market such that traded value for fee purposes in USDT is less than 10^6
+* During the epoch 2 let the traded value be greater than 10^6
+* During epoch 3, setup and fund multiple reward account for the market ETHUSDT with a different party:
+    * Transfer 10000 $VEGA to `ETHUSDT | market creation | $VEGA` 
+
+#### Expectation
+At the end of epoch 2, 10000 VEGA rewards should be distributed to the proposer of the ETHUSDT market, bringing their general `USDT` balance to 10000.
+The reward account balance should be empty.
+
+Then, at the end of epoch 3, the 10000 VEGA rewards should again be distributed to the proposer of ETHUSDT, bringing their general `USDT` balance to 20000.
+The reward account balance should again be empty
+
+Then, at the end of epoch 4, no further VEGA rewards should be distributed, the proposer of ETHUSDT's general `USDT` balance should stay at 20000.
+The reward account balance should still be empty, as there were no eligible markets so no transfer should occur.
+
+
+### Distributing market creation rewards - Multiple markets eligible, one already paid (<a name="0056-REWA-046" href="#0056-REWA-046">0056-REWA-046</a>)
+#### Rationale 
+A market reward pool funded with the same asset by the same party with different market scopes should pay to all markets even if already paid
+#### Setup
+* Setup a market ETHUSDT settling in USDT.
+* Setup a market BTCDAI settling in DAI with a different party.
+* The value of `marketCreationQuantumMultiple` is `10^6` and `quantum` for both `USDT` and `DAI` is `1`. 
+* Setup and fund recurring reward account transfers using the market_proposer metric and `USDT` metric asset:
+    * Transfer 10000 $VEGA to `ETHUSDT | market creation | $VEGA` 
+* start trading in the market such that traded value for fee purposes in USDT is less than 10^6
+* During epoch 2 let the traded value on ETHUSDT and BTCUSDT be greater than 10^6
+* During epoch 3, setup and fund recurring reward account transfers using the market_proposer metric and `USDT` metric asset:
+    * Transfer 10000 $VEGA to `all | market creation | $VEGA` 
+
+#### Expectation
+At the end of epoch 2, 10000 VEGA rewards should be distributed to only the ETHUSDT creator.
+    The general account balance of the ETHUSDT creator should be 10000.
+    The general account balance of the BTCDAI creator should be 0.
+    The reward pool balance should be 0.
+
+At the end of epoch 3, 10000 VEGA should be distributed split between the BTCDAI creator and the ETHUSDT creator.
+    The general account balance of the ETHUSDT creator should be 15000.
```

Comment:
I don't think this is correct. As you're setting the metric asset in this example, it means that it is restricted to markets settling in USDT, which means again only ETHUSDT gets the whole lot, meaning they would have a balance of 20k and poor BTCDAI would have a balance of 0.

Reply:
Fixed up + added a new test